doc/decisions/0_drafts/errors_core.md

```diff
@@ -0,0 +1,292 @@
+# Errors without `errorKey`
+
+## Problem
+
+We often use a `Key * errorKey` argument to return error information via metadata (see related decisions).
```

Comment:
Either directly link (if the decisions must be known to understand this decision) or it is enough if related decisions are linked below in its section.

```suggestion
We often use a `Key * errorKey` argument to return error information via metadata.
```